2.train-VAE/betatcvae.py

```diff
@@ -321,7 +307,7 @@ def extract_latent_dimensions(model, data_loader, metadata, path):
     latent_space = []
     with torch.no_grad():
         for batch in data_loader:
-            data = batch[1]
+            data = batch[0]
```

Comment:
why a switch here?

Reply:
Adjusted hidden dimensions earlier so that this code functioned meant I had to adjust here too!